sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusOptions.cs

```diff
@@ -146,6 +146,25 @@ public int MaxConcurrentSessions
         /// </summary>
         public int MaxMessageBatchSize { get; set; } = 1000;
 
+        /// <summary>
+        /// Gets or sets the minimum number of events desired for a batch. This setting applies only to functions that
+        /// receive multiple events. This value must be less than <see cref="MaxMessageBatchSize"/> and is used in
+        /// conjunction with <see cref="MaxWaitTime"/>. If <see cref="MaxWaitTime"/> passes and less than <see cref="MinMessageBatchSize"/>
+        /// has been received, the function will be invoked with a partial batch. Default 1.
+        /// </summary>
+        public int MinMessageBatchSize { get; set; } = 1;
+
+        /// <summary>
+        /// Gets or sets the maximum time that the trigger should wait to fill a batch before invoking the function.
+        /// This is only considered when <see cref="MinMessageBatchSize"/> is set to larger than 1 and is otherwise unused.
+        /// If less than <see cref="MinMessageBatchSize" /> events were available before the wait time elapses, the function
+        /// will be invoked with a partial batch. This value should be no longer then 50% of the longest message lock duration;
+        /// for example, the LockDuration Property. Therefore, the maximum allowed value is 2 minutes and 30 seconds.
+        /// Otherwise, you may get lock exceptions when messages are pulled from the cache.
+        /// The default value is 30 seconds.
+        /// </summary>
```

Comment:
```suggestion
        /// <summary>
        /// Gets or sets the maximum time that the trigger should wait to fill a batch before invoking the function.
        /// This is only considered when <see cref="MinMessageBatchSize"/> is set to larger than 1 and is otherwise unused.
        /// If less than <see cref="MinMessageBatchSize" /> messages were available before the wait time elapses, the function
        /// will be invoked with a partial batch. This value should be no longer then 50% of the entity message lock duration.
        /// Therefore, the maximum allowed value is 2 minutes and 30 seconds.
        /// Otherwise, you may get lock exceptions when messages are pulled from the cache.
        /// The default value is 30 seconds.
        /// </summary>
```